react/ManualPrice.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { FunctionComponent, Fragment, useState } from 'react'
+import { FormattedMessage, useIntl } from 'react-intl'
+import { FormattedPrice } from 'vtex.formatted-price'
+import { Button, ButtonPlain, InputCurrency, Modal, Tag } from 'vtex.styleguide'
+import { useRuntime } from 'vtex.render-runtime'
+
+import { useItemContext } from './ItemContext'
+import { getFormattedPrice } from './utils/price'
+
+const ManualPrice: FunctionComponent = () => {
+  const {
+    item,
+    itemIndex,
+    shouldAllowManualPrice,
+    onSetManualPrice,
+  } = useItemContext()
+  const [isModalOpen, setIsModalOpen] = useState(false)
+  const [manualPrice, setManualPrice] = useState(item.sellingPrice)
+  const {
+    culture: { currency, locale },
+  } = useRuntime()
+  const intl = useIntl()
+  const priceChanged = item.price !== item.sellingPrice
+
+  const submitManualPrice = (evt: React.FormEvent<HTMLFormElement>) => {
+    evt.preventDefault()
+    onSetManualPrice(manualPrice!, itemIndex)
+    setIsModalOpen(false)
+  }
+
+  const handleManualPriceChange = (
+    evt: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    const newManualPrice = evt.target.value
+    setManualPrice(parseFloat(newManualPrice) * 100)
+  }
+
+  const handleOpenModal = () => {
+    setIsModalOpen(true)
+  }
+
+  const handleCloseModal = () => {
+    setManualPrice(item.sellingPrice)
+    setIsModalOpen(false)
+  }
+
+  const revertToOriginalPrice = () => {
+    onSetManualPrice(item.price!, itemIndex)
+    setManualPrice(item.price)
+    setIsModalOpen(false)
+  }
+
+  return (
+    <Fragment>
+      {shouldAllowManualPrice && (
+        <div className="flex flex-column items-center mt3">
+          {priceChanged ? (
+            <div>
+              <div className={`flex-grow-0 tc mb2 c-muted-1`}>
```

Comment:
you don't need these curly braces here 👀 
```suggestion
              <div className="flex-grow-0 tc mb2 c-muted-1">
```